examples/lock-app/infineon/psoc6/src/AppTask.cpp

```diff
@@ -278,9 +221,6 @@ CHIP_ERROR AppTask::Init()
         numberOfHolidaySchedules = 10;
     }
 
-    chip::DeviceLayer::PlatformMgr().UnlockChipStack();
-
-    // err = LockMgr().Init(state, maxCredentialsPerUser, numberOfSupportedUsers);
     err = LockMgr().Init(state,
```

Comment:
This can now race against `LockMgr().SetCallbacks()`.  Is that safe?  Or should that line have stayed here right after `Init`?

Reply:
I have updated the code to invoke `LockMgr().SetCallbacks(ActionInitiated, ActionCompleted);` after     `err = LockMgr().Init(...)`